app/assets/javascripts/tylium/modules/sidebar.js

```diff
@@ -6,50 +6,55 @@
     this.$viewContent = $sidebar.siblings('[data-behavior~=view-content]');
     this.storageKey = $sidebar.data('storage-key');
 
+    // Must match $sidebarCollapsed and $sidebarExpanded values from tyluim/variables.scss
+    this.collapsedWidth = '6rem';
+    this.expandedWidth =  '15.5rem';
+
     this.init();
   }
 
   Sidebar.prototype = {
     init: function() {
-      this.toggle(this.isSidebarOpen(), 'no-animation');
+      this.toggle(this.isSidebarOpen());
 
       var that = this;
-      $('[data-behavior~=sidebar-toggle]').on('click', function() {
-        if (that.isSidebarOpen() && $(this).is('[data-behavior~=open-only]')) return;
 
-        that.toggle(!that.isSidebarOpen(), 'animate');
+      $('[data-behavior~=sidebar-toggle]').on('click', function(e) {
+        if ((that.isSidebarOpen() && $(this).is('[data-behavior~=open-only]')) || ($(e.target).data('behavior') == 'add-node')) return;
+
+        that.toggle(!that.isSidebarOpen());
       });
     },
     changeState: function(state) {
       localStorage.setItem(this.storageKey, state);
       Turbolinks.clearCache();
     },
-    close: function(animationClass) {
-      this.$navbar.css('left', '0px');
+    close: function() {
+      this.$navbar.css('left', '0rem');
       this.$sidebar
-        .removeClass('sidebar-expanded no-animation animate')
-        .addClass('sidebar-collapsed ' + animationClass)
-      this.$viewContent.css({'left': '43px', 'width': 'calc(100vw - 43px)'});
+        .removeClass('sidebar-expanded')
+        .addClass('sidebar-collapsed')
+      this.$viewContent.css({'left': this.collapsedWidth, 'width': 'calc(100vw - ' + this.collapsedWidth + ')'});
 
       this.changeState(false);
     },
     isSidebarOpen: function() {
       return JSON.parse(localStorage.getItem(this.storageKey))
     },
-    open: function(animationClass) {
-      this.$navbar.css('left', '207px');
+    open: function() {
+      this.$navbar.css('left', '9.5rem');
```

Comment:
Can we name this magic number? Example:
```
this.navbarOffset = '9.5rem';
...
this.$navbar.css('left', this.navbarOffset);
```

Reply:
This number is the result of subtracting the collapsed width from the expanded width. 
I went ahead and changed this to use a calculated value as this would need to be adjusted if the widths are changed in the future. This change now eliminates the need to adjust this value manually.